app/pom.xml

```diff
@@ -145,8 +145,8 @@
 
         <!-- Kafka Connect -->
         <dependency>
-            <groupId>org.apache.kafka</groupId>
-            <artifactId>connect-api</artifactId>
+            <groupId>io.quarkus</groupId>
```

Comment:
I do not see the relation between this and the PR purpose. Changing from standard apache libraries to quarkus ones for better native compatibility?

Reply:
Agreed - and isn't Kafka Connect definitely different from the quarkus kafka client?

Reply:
it's different yes, but looks like we are using some APIs from kafka clients, and this was required in order to compile and with `connect-json` (just below) we still have the kafka-connect api